samples/sample_extension/test_sample_synchronous_extension.dart

```diff
@@ -7,45 +7,44 @@ library test_sample_extension;
 import 'sample_synchronous_extension.dart';
 
 void check(bool condition, String message) {
-  if (!condition) {
-    throw new StateError(message);
-  }
+  if (!condition) throw StateError(message);
+  print('$message - passed');
```

Comment:
I added a print statement just because it felt reassuring to get _some_ output :)